src/ansys/geometry/core/connection/conversions.py

```diff
@@ -662,3 +662,21 @@ def trimmed_surface_to_grpc_trimmed_surface(
         v_min=trimmed_surface.box_uv.interval_v.start,
         v_max=trimmed_surface.box_uv.interval_v.end,
     )
+
+
+def line_to_grpc_line(line: Line) -> GRPCLine:
+    """Convert a ``Line`` to a line gRPC message.
```

Comment:
```suggestion
    """Convert a line to a line gRPC message.
```
I don't think in the method description we need to tag line as a code entity.